examples/data-sources/hpegl_vmaas_pool_member_group/data-source.tf

```diff
@@ -0,0 +1,5 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_pool_member_group" "tf_pool_group" {
```

Comment:
Can you name it better ? it should not be understand as data source for Group. Name it as hpegl_vmaas_lb_pool_member_group

Reply:
done